packages/server/src/api/routes/tests/viewV2.spec.ts

```diff
@@ -525,4 +543,475 @@ describe.each([
       expect(row.Country).toEqual("Aussy")
     })
   })
+
+  describe.only("row operations", () => {
```

Comment:
Remove:
```suggestion
  describe("row operations", () => {
```

Reply:
The shame. I might write a lint for this.